Assets/Scripts/Health.cs

```diff
@@ -4,10 +4,13 @@ public class Health : MonoBehaviour
 {
     [SerializeField] int max = 3;
     int _current;
+    Animator _animator;
+    static readonly int AnimatorHealth = Animator.StringToHash("health");
```

Comment:
since we need to handle both animation and sound on death and maybe other things in the future like restarting level etc. I think we could do that through events like I did here: [pr](https://github.com/FrozzLab/2023_11_eng_03_8week/pull/12/files#diff-8ef224c7b24d00280884e92e6b7f674d496e370f95e0909fb7fb5e89eed74209)

but since it's not merged yet, feel free to merge yours and we can adjust it with another PR. Just let me know if you agree

Reply:
Yes having a death event would be way better. To be honest I just didn't know how to do events in Unity. Will refactor in the future.